### PR TITLE
Fluff computers 2.0

### PR DESCRIPTION
**Pull request by jakmak3**
old PR --> https://github.com/d3athrow/vgstation13/pull/12575

This adds in several new computers that do litterally nothing except give a access denied message. The only purpose for these computers is for vaults/away missions for immhulsions purposes. When jwmturners mapping PR is closed/merged i will make a pr to replace some of the empty computer consoles in the white ship with these.

Github broke so i had to re make the PR. essentially nothing has changed and i fixed some of the stuff the reviewers had pointed out to me.

### CODE REVIEW

**Comment by ghost** [comment]
if i had a dollar for every "fuck github" commit and branch name...

**Comment by Intigracy** [comment]
https://xkcd.com/1597/

**Review by PJB3005 — CHANGES_REQUESTED** [review]
You still need to resolve all the issues in the previous PR, just so you know.

File: code/game/machinery/computer/fluffcomputer.dm [hunk 1]
```diff
@@ -0,0 +1,141 @@
+/* this entire thing is a non player operatable computer that just gives a access denied message or something like that when you try to use it.
+It does nothing and only serves to look good and for immhulsions. intended for vaults away missions and things of that sort.
+This is not related to actual functional consoles in any way */
+
+/obj/machinery/computer/unusable_communication_relay
+    name = "Communications Relay"
+    desc = "A code locked console that you do not know how to use. Probably doesnt do anything interesting."
```
Comment (CptWad) [inline code/game/machinery/computer/fluffcomputer.dm]: ...doesn't do anything of interest."

File: code/game/machinery/computer/fluffcomputer.dm [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_communication_relay/attack_hand(var/mob/user as mob)
+    say("Access Denied")
+
+
+/obj/machinery/computer/unusable_communication_relay/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works with NT technology ")
+
+/obj/machinery/computer/unusable_shuttle_control_1 /*fluff shuttle console 1 */
+    name = "Shuttle Control"
+    desc = "A code locked console that controls the ship. Doesnt look like you will be flying the ship anytime soon."
```
Comment (CptWad) [inline code/game/machinery/computer/fluffcomputer.dm]: Doesn't

File: code/game/machinery/computer/fluffcomputer.dm [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+/obj/machinery/computer/unusable_shuttle_engine_control/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works with NT technology ")
+
+
+ /* the next 2 are for the white ship */
+
+/obj/machinery/computer/whiteship_console_1  /*console 1*/
+    name = "Starmap"
+    desc = "A console with a map of the local area. Just by looking at this thing you can tell it is years out of date and is too old to be used effectivly"
```
Comment (CptWad) [inline code/game/machinery/computer/fluffcomputer.dm]: Remove effectively. "too old to be used" is good enough

File: code/game/machinery/computer/fluffcomputer.dm [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    icon_state = "comm_serv"
+    circuit = "/obj/item/weapon/circuitboard/whiteship_console_1"
+
+    light_color = LIGHT_COLOR_GREEN
+
+/obj/machinery/computer/whiteship_console_1/attack_ai(var/mob/user as mob)
+        to_chat(user, "The console is so old that it cannot be remotely controlled by silicons.")
```
Comment (CptWad) [inline code/game/machinery/computer/fluffcomputer.dm]: [this] console is so old

File: code/game/machinery/computer/fluffcomputer.dm (hunk 3 above)
Comment (Shadowmech88) [reply code/game/machinery/computer/fluffcomputer.dm]: Also lacking a period.

File: code/game/machinery/computer/fluffcomputer.dm (hunk 4 above)
Comment (Intigracy) [reply code/game/machinery/computer/fluffcomputer.dm]: [this] doesn't do anything though, and neither does \this if you meant that.

Comment (Intigracy) [reply code/game/machinery/computer/fluffcomputer.dm]: Though` The` should be changed to `This`

Comment (CptWad) [inline code/game/machinery/computer/fluffcomputer.dm]: yes replace the with this thats what I meant

File: code/game/machinery/computer/fluffcomputer.dm [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_communication_relay
+    name = "Communications Relay"
+    desc = "A code locked console that you do not know how to use. Probably doesn't do anything of interest."
+    icon_state = "comm"
+    circuit = "/obj/item/weapon/circuitboard/unusable_communication_relay"
+
+    light_color = LIGHT_COLOR_BLUE
+
+/obj/machinery/computer/unusable_communication_relay/attack_ai(var/mob/user as mob)
+        to_chat(user, "<span class='warning'>The systems firewall prevents you from accessing it.</span>")
```
Comment (Exxion) [inline code/game/machinery/computer/fluffcomputer.dm]: `system's`

File: code/game/machinery/computer/fluffcomputer.dm [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_shuttle_control_1 /*fluff shuttle console 1 */
+    name = "Shuttle Control"
+    desc = "A code locked console that controls the ship. Doesn't look like you will be flying the ship anytime soon."
+    icon_state = "shuttle"
+    circuit = "/obj/item/weapon/circuitboard/unusable_shuttle_control_1"
+
+    light_color = LIGHT_COLOR_CYAN
+
+/obj/machinery/computer/unusable_shuttle_control_1/attack_ai(var/mob/user as mob)
+        to_chat(user, "<span class='warning'>The systems firewall prevents you from accessing it.</span>")
```
Comment (Exxion) [inline code/game/machinery/computer/fluffcomputer.dm]: `system's`

File: code/game/machinery/computer/fluffcomputer.dm [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_shuttle_control_2 /*fluff shuttle control 2 */
+    name = "Shuttle Control"
+    desc = "A code locked console that controls the ship. Doesn't look like you will be flying the ship anytime soon."
+    icon_state = "syndishuttle"
+    circuit = "/obj/item/weapon/circuitboard/unusable_shuttle_control_2"
+
+    light_color = LIGHT_COLOR_RED
+
+/obj/machinery/computer/unusable_shuttle_control_2/attack_ai(var/mob/user as mob)
+        to_chat(user, "<span class='warning'>The systems firewall prevents you from accessing it.</span>")
```
Comment (Exxion) [inline code/game/machinery/computer/fluffcomputer.dm]: `system's`

File: code/game/machinery/computer/fluffcomputer.dm [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_shuttle_engine_control /*fluff shuttle console 3 */
+    name = "Engine Control"
+    desc = "A code locked console that controls the ship's engines and power systems."
+    icon_state = "airtunnel01"
+    circuit = "/obj/item/weapon/circuitboard/unusable_shuttle_engine_control"
+
+    light_color = LIGHT_COLOR_RED
+
+/obj/machinery/computer/unusable_shuttle_engine_control/attack_ai(var/mob/user as mob)
+        to_chat(user, "<span class='warning'>The systems firewall prevents you from accessing it.</span>")
```
Comment (Exxion) [inline code/game/machinery/computer/fluffcomputer.dm]: `system's`

**Review by Exxion — CHANGES_REQUESTED** [review]

File: code/game/machinery/computer/fluffcomputer.dm [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+/obj/machinery/computer/unusable_shuttle_control_2/attack_hand(var/mob/user as mob)
+    say("Access Denied")
+
+/obj/machinery/computer/unusable_shuttle_control_2/emag(mob/user as mob)
+    to_chat(user, "You hold the cryptographic sequencer up to the ID scanner and nothing happens. Guess this only works with NT technology")
+
+
+/obj/machinery/computer/unusable_shuttle_engine_control /*fluff shuttle console 3 */
+    name = "Engine Control"
+    desc = "A code locked console that controls the ship's engines and power system's."
```
Comment (Exxion) [inline code/game/machinery/computer/fluffcomputer.dm]: This one was correct to begin with

**Review by Exxion — CHANGES_REQUESTED** [review]

**Comment by 9600bauds** [comment]
[Needs Grammar/Spelling)

**Pull request closed without merging**